smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/PackageJsonGenerator.java

```diff
@@ -60,7 +60,7 @@ static void writePackageJson(
         ObjectNode devDeps = node.getObjectMember("devDependencies").orElse(Node.objectNode());
         if (devDeps.containsMember(TypeScriptDependency.VITEST.packageName)) {
             ObjectNode scripts = node.getObjectMember("scripts").orElse(Node.objectNode());
-            scripts = scripts.withMember("test", "vitest run");
+            scripts = scripts.withMember("test", "vitest run --passWithNoTests");
```

Comment:
We should figure out where `vitest` is being imported in our use cases.

If not being used anywhere, maybe we can remove it from `TypeScriptDependency`.